### DISCUSSION TITLE
Best practices using with NuxtContent?

### DISCUSSION THREAD

**Discussion by narr07** [s00d/nuxt-i18n-micro#65] [Q&A] [answered]
This module can use with nuxt-content?

**Comment by s00d** [comment]
I haven't studied this issue myself, but I saw that someone integrated it. Unfortunately, I can't give you any advice as I don't really understand how to use it in practice.

**Comment by s00d** [comment, marked as answer]
I have added an integration example with Nuxt Content:

[Nuxt Content Integration Example](https://github.com/s00d/nuxt-i18n-micro/tree/main/test/fixtures/content)

I should mention upfront that I intentionally created general pages for the tests, meaning a single `.md` file is used for all locales. In a real project, this might not be necessary. It is likely sufficient to create just a couple of components for locale switching and text rendering from translations. You can find examples of such components here: 

[Nuxt Components for Localization](https://github.com/s00d/nuxt-i18n-micro/tree/main/test/fixtures/content/components/content)

**Reply by narr07** [reply]
Thank you very much, I was very helped by the code you provided
